nomad-cluster-security.md

```diff
@@ -0,0 +1,14 @@
+# Nomad Cluster Security
+
+1. What is the noamd cluster size?
+1. Is Nomad deployed single region or multi-region? What is the deployment strategy? How is the high availability and failover designed? How is leader and follower selected?
+1. Is Nomad integrated with consul? How the DNS resolutions works to connect to the consul cluster?
```

Comment:
Please clarify (and fix grammar) the second question, in particular, if the answer to the first one would be "no", does that change anything about the second?

Reply:
Fixed the grammar error with the help of google docs  and if the answer to the first one would be "no" , it doesn't change anything about the second .